utils/wasm-gen/src/lib.rs

```diff
@@ -282,6 +286,11 @@ struct WasmGen<'a> {
     calls_indexes: Vec<FuncIdx>,
 }
 
+enum GearStackEndExportSeed {
```

Comment:
You can use Option<u32> for such cases. But it's not a big problem

Reply:
Agree, `Option<u32>` allows avoiding introducing a new type.

Reply:
@grishasobol @shamilsan Sure it can be used. But I used created a new type for better readability, 'cause  GearStackEndExportSeed is more constructive then Option<u32>. It is also not effect to performance